### ISSUE TITLE
Allow different input types

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#398]
Blocked by #396. 

We should allow the UI to add different input types, depending on the operation which is selected.
For example GetElementsInRange would have an input type of Pair rather than just entity seed. The form for adding pairs will be different to the form for regular seeds.

This will allow for the support of a wider variety of operations.

**Comment by p013570** [comment]
Merged into develop.
